src/content/docs/usage/Animating/procedural.mdx

```diff
@@ -9,125 +9,156 @@ import sophGraphImg from 'assets/bopGraph.png';
 
-1. Time Node exposes <a href="/docs/usage/node-graph/nodes/animation/time#normalized-time">Normalized Time</a> - which is a value between 0 and 1. Where 0 is start of the animation and 1 is end of it. For 2 second animation, normalized time at 1 second will equal `0.5`.
-2. We are connecting Normalized Time to Image's Opacity. So at the beginning of the animation, the image will be transparent (`0` opacity) and fully opaque at the end (`1` opacity).
+2. Create a smoothing effect by connecting the time to Easing node. For example, use Elastic easing for a bounce effect.
 
-## Let's make something slightly more sophisticated.
+3. Combine rotation and scaling:
+    - Scale: Connect the eased time directly. This makes the object grow from invisible (0) to full size (1).
+    - Rotation: Multiply the eased time by 360 so the object rotates one full turn.
+
+4. Combine effects using a Matrix Input Node. This allows scale and rotation to be applied together.
+Your node connection should look like the ones in the image below
+  <Image src={sophGraphImg} alt="Simple graph that connects Normalized Time of Time Node to Layer's Opacity input"/>
+
+<br/>
 
-<Image src={sophGraphImg} alt="Simple graph that connects Normalized Time of Time Node to Layer's Opacity input"/>
+The result is a smooth, rotating, and scaling animation, perfect for attention-grabbing effects.
 
-The core concept in this graph is:
+  <video src={bopVideo} muted autoplay loop/>
 
-1. Get the time of the animation (0 - 1 range)
-2. Create a smoothing effect (elastic in that case) by plugging our time to Easing node. Read more about <a href="/docs/usage/node-graph/nodes/animation/easing">easings here</a>.
-3. Create a rotation + scale effect by combining two nodes with "Matrix Input" and "Matrix Output"
-    - Scale can be directly connected with easing, because we want it to go from invisible (0 scale) to full size (1 scale)
-    - Rotation must be in 0 - 360 range to make image rotate in a full circle. This is done by multiplying eased time by 360.
-4. At last, we are plugging our source image to the Scale Node. In our case Scale Node not only scales but also rotates, because we plugged in the result of rotation to it. You can combine multiple effects into one with yellow (<a href="/docs/usage/node-graph/property-sockets/#-matrix3x3">Matrix</a>) inputs.
 
-And the effect:
+<br/>
+
+## Key takeaways
+
+- Procedural animations give you fine control over timing and transitions.
+- Experimentation is key. Try combining nodes in creative ways.
+- The more comfortable you get with nodes, the faster you can create unique, dynamic effects.
+
+<br/>
 
-<video src={bopVideo} muted autoplay loop/>
+## Further learning
+To learn more:
+- (<a href="/docs/usage/node-graph/nodes/animation/easing">Easing</a>) 
+- (<a href="/docs/usage/node-graph/nodes/animation/time#normalized-time">Normalized time</a>)  
+- (<a href="/docs/usage/node-graph/property-sockets/#-matrix3x3">Matrix</a>) 
```

Comment:
Unnecessary `<br/>` between Key takeaways and Further learning. 

Additioanlly adding `(` and `)` on each bullet point looks weird 

<img width="806" height="469" alt="Image" src="https://github.com/user-attachments/assets/b48af012-282e-4024-bca9-2deb2137c21b" />